lib/radioactive_bunnies/worker.rb

```diff
@@ -1,17 +1,14 @@
 require 'atomic'
 require 'radioactive_bunnies/context'
-
+require 'radioactive_bunnies/deadletter_worker'
 module RadioactiveBunnies::Worker
 
-  def ack!
```

Comment:
I can definitely see that. If we go that route, would you go the route of overriding in the service-level RpcWorker to accept a payload? We dropped it since the main (current) implementer returns false via the meta'd error methods or it gives a hash payload back to transform.